gtest/src/manager.rs

```diff
@@ -221,8 +222,9 @@ impl ExtManager {
     #[track_caller]
```

Comment:
Track caller may be also removed here to highlight correct "source" of panic

Reply:
Yes, not sure how I missed it

Reply:
Fixed